tests/test_unit/test_napari_plugin/test_poses_loader_widget.py

```diff
@@ -167,3 +168,28 @@ def test_on_load_clicked_with_valid_file_path(
     # Check that a Points layer was added to the viewer
     points_layer = poses_loader_widget.viewer.layers[0]
     assert points_layer.name == f"poses: {file_path.name}"
+
+
+def test_dimension_slider_matches_frames(make_napari_viewer_proxy):
```

Comment:
Having this test is good for my sanity, let's keep it in, even if it tests "someone else's code".